.github/workflows/push.yml

```diff
@@ -3,10 +3,12 @@ name: Node CI
 on:
   push:
     branches:
-    - master
+    - main
+    - next
   pull_request:
     branches:
-    - master
+    - main
+    - next
```

Comment:
Sorry, I think I cloned Storybook's fork of this project. Some of these changes are unrelated. But check out the snapshot below :)